README.md

```diff
@@ -5,14 +5,46 @@
 **Maintainer:** [@marianunez](https://github.com/marianunez)
 **Co-Maintainer:** [@larkox](https://github.com/larkox)
 
-This plugin adds custom attributes to users in your Mattermost instance.
-
-Currently the plugin only exposes the specified attributes in the user profile popover, but this plugin can be extended to allow displaying attributes elsewhere in the user interface, such as badges next to usernames.
+This plugin adds custom attributes to users in your Mattermost instance.  You can specify an Attribute, and then specify specific users or groups of users that display that attribute on their public profile so other users can identify them easily.  This can be useful when there are Team Leads, Timezones, etc. and makes it easy to show who is on a particular team or Project.  
+Currently the plugin only exposes the specified attributes in the user profile popover, but this plugin could be extended to allow displaying attributes elsewhere in the user interface, such as badges next to usernames. 
 
 We use this plugin on https://community.mattermost.com to distinguish Mattermost core committers and staff.
 
 ![image](https://user-images.githubusercontent.com/13119842/58710612-b5c7b380-838a-11e9-9974-4487daf82da5.png)
 
+# Version 1.20 (Latest)
+
+## Installation
+
+Install via Plugin Marketplace (Recommended)
+ - Go to Main Menu > Plugin Marketplace in Mattermost
+ - Search for "Custom Attribute" or manually find the plugin from the list and click **Install**
+ - After the plugin has downloaded and been installed, click the **Configure** button
+ - Set **Enable Plugin** to **True**, then click "Save" at the bottom of the screen
+ - Proceed to Configure your Attribute rules
+ 
+ ## Configuration
+ 
+Before you start, Identify the attributes you want to display on a user's profile popover.  These can contain emojis. Some examples could be "Timezone:PST", "Development Team", "Executive Team Member", "Mentor", etc. then Identify the groups or particular usernames that should display those atrributes.  A spreadsheet can help to organize things.
+
+1. Click "Add Custom Attribute" button, a text box will appear.  Add text and/or emojis to the field.  This text will be displayed on a user's profile.
```

Comment:
The attribute text actually support markdown so it could also be a link. So maybe we can rephrase it to something like (only a suggestions, reword as needed) 
```suggestion
1. Click "Add Custom Attribute" button, a text box will appear.  Add the text that would appear in the user's profile popover. The text supports markdown and could include emojis and/or links, i.e. "[Integrations Team](https://developers.mattermost.com/internal/rd-teams/#integrations-team)"
```